include/envoy/common/platform.h

```diff
@@ -84,6 +94,85 @@ struct msghdr {
   int msg_flags;
 };
 
+// Posix structure for tcp info.
+// See https://github.com/torvalds/linux/blob/master/include/uapi/linux/tcp.h
+// Some of them may remain uninitialized. If you are using this struct please refer to
+// the implementation of Windows to see that the metric you are querying for is supported.
```

Comment:
refer to the Windows SIO_TCP_INFO documentation

Reply:
Sure I can add the pointer to the Windows documentation but anyone would still need to go though the Windows implementation to verify that we have implemented the mapping from SIO_TCP_INFO -> TCP_INFO.

It might be there is a kernel update for SIO_TCP_INFO and Envoy implementation has not picked up the new version.

Reply:
Oh yeah I see. Looking at the Windows OsSysCallsImpl::socketTcpInfo makes more sense and gives a faster, concrete answer

Reply:
Added a link to the SIO_TCP_INFO documentation.